docs/release-notes/NuGet-5.5.md

```diff
@@ -0,0 +1,118 @@
+
+* --skip-duplicate support for dotnet nuget push - [#8778](https://github.com/NuGet/Home/issues/8778)
+
+* msbuild /restore works with Packages.Config - [#8506](https://github.com/NuGet/Home/issues/8506)
+
+**DCR:**
+
+* nuget.exe pack should no longer warn about packing a SemVer 2.0.0 package - [#5201](https://github.com/NuGet/Home/issues/5201)
+
+* Embedded Icons support in NuGet PackageManager UI - [#8189](https://github.com/NuGet/Home/issues/8189)
+
+* Logic in _GetRestoreProjectStyle should be in a task - [#8804](https://github.com/NuGet/Home/issues/8804)
+
+* Deprecation info is not visible, by default, on installed tab - [#8541](https://github.com/NuGet/Home/issues/8541)
+
+**Bug:**
+
+* Rework Self-Updater with V3 Apis - [#4197](https://github.com/NuGet/Home/issues/4197)
+
+* Wrong package dependency version If package dependency version is set to '*' - [#6697](https://github.com/NuGet/Home/issues/6697)
+
+* ErrorUnsafePackageEntry error message is not pointing to source of problem - [#7505](https://github.com/NuGet/Home/issues/7505)
+
+* dotnet restore failing with TaskCanceledException - [#7842](https://github.com/NuGet/Home/issues/7842)
+
+* Lock file is not honored in "*" scenarios  - [#8073](https://github.com/NuGet/Home/issues/8073)
+
+* NuGet.exe does not resolve to the latest version of a package when using * in PackageReference (MSBuild/Dotnet/VS restore do) - [#8432](https://github.com/NuGet/Home/issues/8432)
+
+* dotnet list package with multi targeting WPF project - [#8463](https://github.com/NuGet/Home/issues/8463)
+
+* Plugin:  "A task was cancelled" - problem with ADO authentication due to this. - [#8528](https://github.com/NuGet/Home/issues/8528)
+
+* Improve ConcurrencyUtilities (reduce CPU usage) - [#8653](https://github.com/NuGet/Home/issues/8653)
+
+* DG Spec for unloaded project scenarios should not be written in preview restores - [#8793](https://github.com/NuGet/Home/issues/8793)
+
+* The Visual Studio NuGet packages (RestoreManagerPackage) needs to auto load on solution build events - [#8796](https://github.com/NuGet/Home/issues/8796)
+
+* [Test Failure] Package icon becomes a default icon after installing ‘Newtonsoft.Json’ with latest versoin - [#8814](https://github.com/NuGet/Home/issues/8814)
+
+* Deadlock in VSSettings init - [#8842](https://github.com/NuGet/Home/issues/8842)
+
+* VisualStudio ToolBox is not populated from a NuGet package if a project is placed in a solution folder - [#8868](https://github.com/NuGet/Home/issues/8868)
+
+* VS:  solution restore perpetually fails due to race condition - [#8881](https://github.com/NuGet/Home/issues/8881)
+
+* Constant "loading.." on installed tab, and "searching <term>.." on updates tab - [#8890](https://github.com/NuGet/Home/issues/8890)
+
+* NuGet client tools strategy with multiple feeds on packages with version floats - [#8992](https://github.com/NuGet/Home/issues/8992)
```

Comment:
You didn't delete it. 
We should fix the tool to ignore the duplicates

Reply:
mentioned about this feedback in NuGet/Client.Engineering/issues/228